webserver/js/index.js

```diff
@@ -11,16 +11,15 @@ let uid_text;
 document.addEventListener('DOMContentLoaded', function() {
     $('select').select2();
 
-    user_select = document.querySelector('#user_user');
+    user_select = $('#user_user');
     value_input = document.querySelector('#amount');
     qrcode = document.querySelector('#qrcode');
     qrcode_hint = document.querySelector('#qrcode_hint');
     uid_text = document.querySelector('#uid');
 
-    user_select.addEventListener('change', update_qr);
-    value_input.addEventListener('change', update_qr);
+    $('select').on('select2:select', update_qr);
```

Comment:
Why use jQuery here when we are sticking with `querySelector` elsewhere?

Reply:
Because my personal goal is to not use jQuery when it's not inevitably required.

Reply:
Uh, your commit *added* a use of jQuery AFAICT...

Reply:
Yep, because we need select2 which needs jQuery. So every code piece that belongs to select2 uses jQuery, everything else doesn't. Or do I misunderstand something?

Reply:
Nope, I missed what select2 is about. Sorry

Reply:
No problem :)